docs/getting-started.md

```diff
@@ -36,10 +36,12 @@ Please refer to [this chart](supported-devices.html) for a full list of supporte
   * Your iGPU
   * Your storage devices (HDD/SSD, eMMC, NVMe, etc.)
   * **Your boardname**
+       * For Example, The Boardname for Asus Chromebook C302CA is Cave
```

Comment:
Why is almost every word capitalized?

It should just be:
For example, the board name for the Asus Chromebook C302CA is CAVE